src/config/index.js

```diff
@@ -25,8 +25,8 @@ const config = new Map();
 
   config.set('apiHost',
     process.env.API_HOST ||
-      NODE_ENV === 'development' ?
-        'https://addons-dev.allizom.org' : 'https://addons.mozilla.org');
+      (NODE_ENV === 'development' ?
+       'https://addons-dev.allizom.org' : 'https://addons.mozilla.org'));
```

Comment:
this could be indented one more space since two space tabs are used everywhere else